develop/plone/functionality/actions.rst

```diff
@@ -365,6 +365,13 @@ You might want to have different actions for different site sections or folders.
 
 * http://plone.293351.n2.nabble.com/Custom-portal-tabs-per-subsection-tp5747768p5747768.html
 
+Custom object and object_buttons actions per portal type
+--------------------------------------------------------
+
+If you need to override/customize an action from `object_buttons` that still uses CMFFormController for a specific `portal_type`, check:
```

Comment:
All inline literals must be wrapped in double backticks in reST. Markdown uses single.

Also avoid "/" in narrative text, using the appropriate conjunction.

```
If you need to override or customize an action from ``object_buttons`` that still uses ``CMFFormController`` for a specific ``portal_type``, check:
```

Also for consistency throughout this file, it would be nice to apply double backticks to all other instances of these three inline literals while you are in there, but not required.

Thank you!

Reply:
Thanks for the review. Indeed I was messing with markdown files before editing this rst.

> Also for consistency throughout this file, it would be nice to apply double backticks to all other instances of these three inline literals while you are in there, but not required.

I think it's better to do this in a new PR.